release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -908,6 +927,24 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-builds-bug-fixes"]
 ==== Builds
 
+* Before this update, if you created a build configuration containing an image change trigger in {product-title} 4.7.x or earlier, the image change trigger might trigger builds continuously.
++
+This issue happened because, with the deprecation and removal of the `lastTriggeredImageID` field from the `BuildConfig` spec for Builds, the image change trigger controller stopped checking that field before instantiating builds. {product-title} 4.8 introduced new fields in the status that the image change trigger controller needed to check, but did not.
++
+With this update, the image change trigger controller continuously checks the correct fields in the spec and status for the last triggered image ID. Now, it only triggers a build when necessary. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2004203[BZ#2004203])
+
+* Before this update, image references in Builds needed to specify the Red Hat registry name explicitly. With this update, if an image reference does not contain the registry, the Build searches the Red Hat registries and other well-known registries to locate the image. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011293[BZ#2011293])
+
+[discrete]
+[id="ocp-4-10-jenkins-bug-fixes"]
+==== Jenkins
+
+* Before this update, version 1.0.48 of the OpenShift Jenkins Sync Plugin introduced a `NullPointerException` error when Jenkins notified the plug-in of new jobs that were not associated with an OpenShift Jenkins Pipeline Build Strategy Build Config. Ultimately, this error was benign because there was no `BuildConfig` object to associate with the incoming Jenkins Job. Core Jenkins ignored the exception in our plug-in and moved on to the next listener. However, a long stack trace showed up in the Jenkins log that distracted users. With this update, the plug-in resolves the issue by making the proper checks to avoid this error and the subsequent stack trace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2030692[BZ#2030692])
+
+* Before this update, performance improvements in version 1.0.48 of the OpenShift Sync Jenkins plug-in incorrectly specified the labels accepted for `ConfigMap` and `ImageStream` objects intended to map into the Jenkins Kubernetes plug-in pod templates. As a result, the plug-in no longer imported pod templates from `ConfigMap` and `ImageStream` objects with the `jenkins-agent` label.
++
+This update corrects the accepted label specification so that the plug-in imports pod templates from `ConfigMap` and `ImageStream` objects that have the `jenkins-agent` label. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2034839[2034839])
```

Comment:
```suggestion
This update corrects the accepted label specification so that the plug-in imports pod templates from `ConfigMap` and `ImageStream` objects have the `jenkins-agent` label. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2034839[2034839])
```

Reply:
Here, we're using `that` to add a restrictive clause. 
- From which `ConfigMap` and `ImageStream` objects does the plug-in import pod templates? 
- Only from `ConfigMap` and `ImageStream` objects that have a `jenkins-agent` label.

Reply:
Here's how I read it: "This update corrects A so that the Bs and Cs from Ds that have the E label." That strikes me as ungrammatical. Or am I just completely not getting it?